recipes/install.rb

```diff
@@ -150,6 +150,10 @@
   action :create
 end
 
+if node['platform_family'].eql?('rhel')
```

Comment:
Can you add a comment that we need them for the Management client (if I remember correctly)? Otherwise, someone will clean them in the future

Reply:
MySQL fails to start as well if we don't have these libraries.